core/src/main/java/org/bitcoinj/crypto/HDPath.java

```diff
@@ -87,7 +85,9 @@ public String toString() {
     private static final InternalUtils.Splitter SEPARATOR_SPLITTER = s -> Stream.of(s.split(SEPARATOR))
             .map(String::trim)
             .collect(Collectors.toList());
-    protected final List<ChildNumber> childNumbers;
+
+    //private final List<ChildNumber> childNumbers;
```

Comment:
This commented out line should be removed.

Reply:
I agree.